cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -3,6 +3,25 @@ import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
 import ResourceListMastheadPo from '@/cypress/e2e/po/components/ResourceList/resource-list-masthead.po';
 import NameNsDescription from '@/cypress/e2e/po/components/name-ns-description.po';
 import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import { KubewardenResourceComplianceReportPagePo } from '@/cypress/e2e/po/extensions/kubewarden/kubewarden.utils';
+
+class ProjectNamespaceDetailPagePo extends PagePo {
+  private static createPath(clusterId: string, id: string) {
+    return `/c/${ clusterId }/explorer/namespace/${ id }`;
+  }
+
+  goTo(clusterId: string, id: string): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(ProjectNamespaceDetailPagePo.createPath(clusterId, id));
+  }
+
+  constructor(clusterId: string, id: string) {
+    super(ProjectNamespaceDetailPagePo.createPath(clusterId, id));
+  }
+
+  kubewardenComplianceReport() {
```

Comment:
This would be rmeoved as part of https://github.com/rancher/dashboard/pull/11668#discussion_r1741987712